modin/pandas/test/test_series.py

```diff
@@ -3512,8 +3504,8 @@ def test_var(data, skipna, ddof):
 
     try:
         pandas_result = pandas_series.var(skipna=skipna, ddof=ddof)
-    except Exception:
-        with pytest.raises(TypeError):
+    except Exception as e:
+        with pytest.raises(type(e)):
```

Comment:
Does pandas now raise a different Exception, or are we just future proofing here?

Reply:
pandas used to throw a `TypeError` now it changed the error type. This makes it so that we check for the same error as pandas, not TypeError.